identity/webapp/src/frontend/hooks/useSendVerificationEmail.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+import axios, { AxiosError } from 'axios';
+
+export enum SendVerificationEmailError { // eslint-disable-line no-shadow
+  UNKNOWN,
+}
+
+type UseSendVerificationEmailMutation = {
+  sendVerificationEmail: () => void;
+  isLoading: boolean;
+  isSuccess: boolean;
```

Comment:
I know it's been used historically in this application, but this is a nasty pattern which we should strive to avoid propagating - can you use a state union like in `useRequestedItems`? It also means you can get rid of the nested ternary in `UnverifiedEmail`

Reply:
Refactored!

Reply:
I wrote booleans for years before I learned The One True Way of making illegal states unrepresentable!